media/js/firefox/os/devices.js

```diff
@@ -197,31 +197,6 @@ if (typeof window.Mozilla === 'undefined') {
     }
     */
 
-    // only if coming from /firefox/os/, detect country
```

Comment:
Note: removing this as part of [bug 1233052](https://bugzilla.mozilla.org/show_bug.cgi?id=1233052) now that the legacy pages are gone.